doc/lua_api.txt

```diff
@@ -1118,54 +1118,109 @@ A box of a regular node would look like:
+An 'octave' is a simple noise generator that outputs a value between -1 and 1.
+The smooth wavy noise it generates has a single characteristic scale, almost
+like a 'wavelength', so on its own does not create fine detail.
+So perlin noise combines several octaves to create variation on multiple scales.
+Each additional octave has a smaller 'wavelength' than the previous, often half
+the size of the previous (if `lacunarity` = 2.0, see below).
+
+This results in noise varying very roughly between -2.0 and 2.0, with an average
+value of 0, so `scale` and `offset` are then used to multiply and offset the
+noise variation. 
+
+The final perlin noise variation is created as follows:
+
+noise = offset + scale * (octave1 +
+                          octave2 * persistence +
+                          octave3 * persistence ^ 2 +
+                          octave4 * persistence ^ 3 +
+                          ...)
+
+(The word 'octave' is used because each additional simple noise generator often
+has a frequency double that of the previous, and in music an 'octave' is a
+doubling of frequency. Noise is analogous to musical tones and harmony and can
+be thought of as spatial music in higher dimensions.)
+
 Noise Parameters
 ----------------
 Noise Parameters, or commonly called "`NoiseParams`", define the properties of
 perlin noise.
 
 ### `offset`
-Offset that the noise is translated by (i.e. added) after calculation.
+After the multiplication by `scale`, this is added the the result and is the
+final step in creating the noise value.
+Can be positive or negative.
 
 ### `scale`
-Factor that the noise is scaled by (i.e. multiplied) after calculation.
+Once all octaves have been combined, the result is multiplied by this.
+Can be positive or negative.
 
 ### `spread`
-Vector containing values by which each coordinate is divided by before
-calculation.
-Higher spread values result in larger noise features.
+For octave1, this is roughly the change of input value needed for a very large
+variation in the noise value generated by octave1. It is almost like a
+'wavelength' for the wavy noise variation.
+Each additional octave has a 'wavelength' that is smaller than the previous
+octave, to create finer detail. `spread` will therefore roughly be the largest
+'wavelength' present in the final noise variation.
 
-A value of `{x=250, y=250, z=250}` is common.
+`spread` is a vector with values for x, y, z to allow the noise variation to be
+stretched or compressed in the desired axes.
+Values are positive numbers.
 
 ### `seed`
-Random seed for the noise. Add the world seed to a seed offset for world-unique
-noise. In the case of `minetest.get_perlin()`, this value has the world seed
-automatically added.
+The 'seed' used by a noise is a random-looking whole number that determines the
+entire pattern of noise variation, like the DNA of a seed that grows into a
+tree. 2 noises with identical 'seed' will create 2 identical patterns of noise
+variation.
```

Comment:
I think we should keep these explanations short but technically correct. Suggestion:
The 'seed' is a often random-looking whole number that determinates the pattern of noise variation. Two identical seeds will result in two identical noise patterns or noise maps.
---
The comparison with DNA is misleading as the tree may look different, depending on the geographic location.

Reply:
Yes it's a vague analogy, i was trying to explain what a seed is, such an explanation could be left out.

Reply:
Removed the analogy and improved this.